app/views/home/businesses.html.erb

```diff
@@ -103,24 +102,24 @@
 
 <div class="border-top py-5">
   <div class="container-lg px-3">
-    <h2 class="alt-h2 mt-6 text-center">You're in good company</h2>
+    <h2 class="alt-h2 mt-6 text-center"><%=t "endosement.title" %></h2>
     <div class="mt-3 pb-6">
       <div class="d-flex flex-wrap flex-items-stretch gut-md">
         <div class="col-md-6">
           <aside class="pquote height-full">
             <img src="https://avatars.githubusercontent.com/caabernathy?s=180" class="pquote-avatar" alt="avatar">
-            Open source aligns with our mission to make the world more open and connected. Working with the community gives us an opportunity to share our work and the challenges we’re aiming to solve, and encourages talented people to contribute their ideas and effort to move the world forward.
+            <%=t "endosement.Christine_Abernathy.message_html" %>
             <p markdown="1" class="pquote-credit">
-          &mdash; Christine Abernathy (<a href="https://github.com/caabernathy">@caabernathy</a>),<br>Open Source at <a href="https://github.com/facebook">Facebook</a>
+            <%=t("endosement.Christine_Abernathy.title_html", id:"caabernathy") %>
```

Comment:
Nice! I'd be tempted to pass in `url: link_to("@caabernathy", "https://github.com/caabernathy"), company_url: link_to("Facebook", "https://github.com/facebook")`. Thoughts? Also: endo**r**sement (missing an `r`).